ktor-server/ktor-server-plugins/ktor-server-call-logging/jvm/test/CallLoggingTest.kt

```diff
@@ -118,8 +120,9 @@ class CallLoggingTest {
             module {
                 install(CallLogging) {
                     format { call ->
-                        "${call.request.uri} -> ${call.response.status()}"
+                        "${call.request.uri} -> ${call.response.status()}, took ${call.processingTimeMillis { 3 }}ms"
```

Comment:
don't we want to have access to the stored `clock`? In this case, a user doesn't have the ability to change the format without changing the default clock. In short, maybe add a getter for clock too?

Reply:
I added default parameter to `processingTimeMillis` so that users don't need to specify it when they use the default one.

Reply:
This test now falls on `Java 8 Windows` and `Java 8 Linux` because `call.processingTimeMillis()` uses not the same clock that was set

Reply:
FIxed